lang/java/reef-common/src/main/java/org/apache/reef/client/DriverConfiguration.java

```diff
@@ -89,11 +88,6 @@
   public static final RequiredImpl<EventHandler<StartTime>> ON_DRIVER_STARTED = new RequiredImpl<>();
 
   /**
-   * This event is fired in place of the ON_DRIVER_STARTED when the Driver is in fact restarted after failure.
-   */
-  public static final OptionalImpl<EventHandler<StartTime>> ON_DRIVER_RESTARTED = new OptionalImpl<>();
```

Comment:
This breaks compatibility. Please send an email to dev@ outlining what is broken, and why we think it is OK. If there are no concerns on the list, we can move forward with this.